td_core/fix_user_pages/fix_it_db_new.py

```
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+"""
+
+from fix_user_pages.fix_it_db_new import work_in_new_tabs_to_db_new
+
+"""
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)